Backpack/build.gradle

```diff
@@ -30,10 +30,6 @@ apply from: "tokens.gradle.kts"
 
 android {
     namespace 'net.skyscanner.backpack'
-    kotlinOptions {
```

Comment:
noticed we were applying these across all modules, rather than the config (plus saw a warning in the logs about using `-Xopt-in` vs `-opt-in`